### ISSUE TITLE
`LDpred2`: Error when filtering characters in chromosome column

### ISSUE THREAD

**Issue by deepchocolate** [comorment/containers#179]
**Describe the bug**
In the presence of characters in the chromosome column of the sumstats file, the process is interrupted by an incorrect access to this column.
https://github.com/comorment/containers/blob/d93d283fac864f94a7f0b45f08bb7cb21609b90a/scripts/pgs/LDpred2/ldpred2.R#L178-L180

**Screenshots**
![image](https://github.com/comorment/containers/assets/16337160/4c050868-8c28-451c-90ad-67768763edae)

**Comment by espenhgn** [comment]
Hi @deepchocolate, are you fixing this problem via https://github.com/comorment/containers/pull/178 already? Or shall I?

**Comment by deepchocolate** [comment]
@espenhgn I haven't fixed it there yet, so if you wish, feel fre to fix :)

**Comment by espenhgn** [comment]
> @espenhgn I haven't fixed it there yet, so if you wish, feel fre to fix :)

Ok, so if I read this correctly, this happens with the character "X" in the "CHR" column, right? I thought you had fixed this (via https://github.com/comorment/containers/pull/131), but alright.

**Comment by deepchocolate** [comment]
> > @espenhgn I haven't fixed it there yet, so if you wish, feel fre to fix :)
> 
> Ok, so if I read this correctly, this happens with the character "X" in the "CHR" column, right? I thought you had fixed this (via #131 closing #138), but alright.

Correct! I think #173 broke it. Unfortunately, I think it is not captured by the tests due to that the chromosome column is lower case in the sumstat files used for testing so the discrepancy between `sumstats$chr` versus `sumstats[, colChr]` is not captured
